pyrage-stubs/pyrage/__init__.pyi

```diff
@@ -37,6 +37,7 @@ class IdentityError(Exception):
 def encrypt(plaintext: bytes, recipients: Sequence[_Recipient]) -> bytes: ...
 def encrypt_file(infile: str, outfile: str, recipients: Sequence[_Recipient]) -> None: ...
 def encrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, recipients: Sequence[_Recipient]) -> bytes: ...
+
 def decrypt(ciphertext: bytes, identities: Sequence[_Identity]) -> bytes: ...
 def decrypt_file(infile: str, outfile: str, identities: Sequence[_Identity]) -> None: ...
-def decrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, recipient: Sequence[_Identity]) -> None: ...
+def decrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, identities: Sequence[_Identity]) -> bytes: ...
```

Comment:
I don't think this return type is right -- `decrypt_io` writes into `out_io`, it doesn't return anything.

Reply:
Ref:

https://github.com/woodruffw/pyrage/blob/0d01c462b9b108ae8dd27791608e5e4c4deb0045/src/lib.rs#L293-L319

Reply:
Thanks for the clarification. Sorry for the delay, I just saw the notification for this.